nilearn/regions/signal_extraction.py

```diff
@@ -270,18 +272,23 @@ def img_to_signals_labels(
         Every point outside the mask is considered
         as background (i.e. no region).
 
-    background_label : number, default=0
-        Number representing background in labels_img.
+    background_label : number, optional
+        Number representing background in labels_img. Default=0.
 
-    order : :obj:`str`, default="F"
-        Ordering of output array ("C" or "F").
+    order : :obj:`str`, optional
+        Ordering of output array ("C" or "F"). Default="F".
 
-    strategy : :obj:`str`, default="mean"
+    strategy : :obj:`str`, optional
         The name of a valid function to reduce the region with.
         Must be one of: sum, mean, median, minimum, maximum, variance,
-        standard_deviation.
+        standard_deviation. Default="mean".
     %(keep_masked_labels)s
```

Comment:
Here we shouldn't see a diff. The default values were moved up to the type definition in another PR so maybe this is a result of incorrect merge conflict resolution